src/geometry/manifold/algebra/structures.lean

```diff
@@ -24,20 +24,16 @@ set_option default_priority 100 -- see Note [default priority]
 
 /-- A smooth semiring is a semiring where addition and multiplication are smooth. -/
```

Comment:
can you adjust this docstring (explaining in particular that in the case of a ring negation is automatically smooth)?